### PR TITLE
feat: support view original inscription

### PR DESCRIPTION
**Pull request by WhiteMinds**
Refs: https://github.com/Magickbase/ckb-explorer-public-issues/issues/528

### CODE REVIEW

File: src/pages/UDT/UDTComp.tsx [hunk 1]
```diff
@@ -166,6 +167,20 @@ export const UDTOverviewCard = ({ typeHash, udt }: { typeHash: string; udt: UDT
         className={styles.cardHeader}
         title={!isMobile && cardTitle}
         hash={typeHash}
+        customActions={[
+          isOmigaInscriptionCollection(udt) && udt.mintStatus === MintStatus.RebaseStart ? (
+            <Tooltip placement="top" title={t('udt.view_original')}>
+              <a
```
Comment (Keith-CY) [inline src/pages/UDT/UDTComp.tsx:173]: Why not use `Link` here

File: src/pages/UDT/index.tsx [hunk 2]
```diff
@@ -1,59 +1,52 @@
...
-  const { search } = useLocation()
+  // The typeHash here could be either udtTypeHash or omigaInscriptionInfoTypeHash.
   const { hash: typeHash } = useParams<{ hash: string }>()
   const { currentPage, pageSize: _pageSize, setPage } = usePaginationParamsInPage()
 
-  const query = new URLSearchParams(search)
-  const filter = query.get('filter')
-  const type = query.get('type')
+  const { filter, view_original } = useSearchParams('filter', 'view_original')
+  const isViewOriginal = view_original === '1'
```
Comment (Keith-CY) [inline src/pages/UDT/index.tsx:27]: Why use a number as `1` instead of a boolean as `true`

**Comment by Keith-CY** [comment]
Please have a review @Daryl-L @PainterPuppets

File: src/pages/UDT/index.tsx (hunk 2 above)
Comment (WhiteMinds) [reply src/pages/UDT/index.tsx:27]: I believe that `0` or `1` representing boolean values in search params is a common convention, while `false` or `true` are less common.

File: src/pages/UDT/UDTComp.tsx (hunk 1 above)
Comment (WhiteMinds) [reply src/pages/UDT/UDTComp.tsx]: This opens in a new tab, so theoretically there's no difference between `link` and `a`.
After thinking about it, using `link` can be more conducive to subsequent maintenance, so I updated it.

File: src/pages/UDT/index.tsx (hunk 2 above)
Comment (Keith-CY) [reply src/pages/UDT/index.tsx:27]: > I believe that `0` or `1` representing boolean values in search params is a common convention, while `false` or `true` are less common.

I would suggest using a semantic field, e.g. `view=original`, to make it extensible. `Omiga` has 2 views, `latest` and `original`, but other inscriptions may have more stages. I'm not sure if there will be more stages, but it's easy to have more views with `view=xxx`

Comment (WhiteMinds) [reply src/pages/UDT/index.tsx:27]: Makes sense, updated.

**Review by Keith-CY — APPROVED** [review]

**Comment by WhiteMinds** [comment]
> * Using search on the `view original inscription` page should stay on the current page and now returns the `Rebase satrt` status page.

Fixed.

> * The `CSV Export` related information at the bottom should be hidden, similar to https://pudge.explorer.nervos.org/inscription/0x1a865222f00d305dde4dae6c6c37c2c6b9a5d5bb2570e2e22e19fb232b655a9d

This feature was temporarily hidden before because the backend interface was still under development. Now that it has been developed, it can be displayed.

> There is a problem with the icon adapting to the mobile terminal. Pay attention to the icon size and margins.

Fixed.

**Review by FrederLu — APPROVED** [review]

**Pull request merged**